packages/taquito/src/batch/rpc-batch-provider.ts

```diff
@@ -152,10 +156,16 @@ export class OperationBatch extends OperationEmitter {
    * @param params Optionally specify the source of the operation
    */
   async send(params?: { source?: string }) {
-    const estimates = await this.estimator.batch(this.operations);
+    const publicKeyHash = await this.signer.publicKeyHash();
+    const publicKey = await this.signer.publicKey();
+    const estimates = await this.estimator.batch(this.operations, { includeRevealOperation: false });
```

Comment:
I see that this change also includes a new `reveal` function in the contract API, should we consider supporting the reveal as an operation that can be added to the batch API?

If that's the case maybe it's not necessary to change this part of the code. However I understand that in the current state we have to change this in order to compensate for the lack of reveal operation support from the batch API.

Reply:
I am not sure of the value added by supporting this as Taquito adds a reveal operation in the batch by itself when needed.